isyfact-standards-doc/src/docs/antora/modules/bausteine/nav.adoc

```diff
@@ -71,4 +71,8 @@
 *** xref:isyfact-jsf-doku:isy-web:konzept/konzept.adoc[Konzept]
 *** xref:isyfact-jsf-doku:isy-web:nutzungsvorgaben/nutzungsvorgaben.adoc[Nutzungsvorgaben]
 
+** Angular
+*** xref:isy-angular-widgets-doc:isy-angular-widgets:konzept/konzept.adoc[Konzept]
```

Comment:
Wenn ihr meinen Empfehlungen in https://github.com/IsyFact/isy-angular-widgets/pull/198 folgt, muss dieser Verweis auch angepasst werden.